### PR TITLE
Tests for login errors verification

### PR DESCRIPTION
**Pull request by justnpT**

### CODE REVIEW

**Comment by justnpT** [comment]
@ludwikkazmierczak  
I'm thinking of creating a ticket to move all getTimeStamp() invocations to the new RandomDataProvider class. This solution removes the dependency that invoking this method currently generates - you can do that only from the basepageobject context. Should I create this ticket ?

File: src/test/java/com/wikia/webdriver/common/dataprovider/RandomDataProvider.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.wikia.webdriver.common.dataprovider;
+
+import java.util.Date;
+
+public class RandomDataProvider {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/dataprovider/RandomDataProvider.java]: MAJOR Add a private constructor to hide the implicit public one. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1118)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/LoginTests.java [hunk 2]
```diff
@@ -7,8 +7,10 @@
 import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.core.drivers.Browser;
 import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.dataprovider.RandomDataProvider;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
 import com.wikia.webdriver.elements.mercury.pages.ArticlePage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/LoginTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/common/dataprovider/RandomDataProvider.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/common/dataprovider/RandomDataProvider.java]: this is not `dataprovider` it doesn't have sense to have additional class which is wrapper for java util function.
Please use `DateTime.now().getMillis()` from `org.joda.time.DateTime`

Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/common/dataprovider/RandomDataProvider.java]: +1

Comment (justnpT) [inline src/test/java/com/wikia/webdriver/common/dataprovider/RandomDataProvider.java]: @ludwikkazmierczak We can create ticket for replacing all getTimestamp() usages with that DateTime.now().getMillis() substitute. Should I do that ?

File: src/test/java/com/wikia/webdriver/elements/mercury/pages/login/SignInPage.java [hunk 4]
```diff
@@ -1,8 +1,14 @@
...
 import com.wikia.webdriver.elements.mercury.components.login.LoginArea;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
 import lombok.Getter;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
 
 /**
  * Created by neptunooo on 06/06/16.
```
Comment (PiotrGackowski) [inline src/test/java/com/wikia/webdriver/elements/mercury/pages/login/SignInPage.java]: Remove "Created" annotation.

**Pull request merged**